### DISCUSSION TITLE
Unique CSS selector for important HTML elements

### DISCUSSION THREAD

**Discussion by egahmad** [laravel/nova-issues#5499] [Ideas & Feature Requests]
We are upgrading to Nova 4 and the custom theming we used to have is not really applied to 4.

The problem is we can't select many element as we used to do in Nova 3

For example the wraper div for the sidebar can't be unbiqly selected, also xpath might change so we also can't select it. 

Other example the wrper div for tables in the list view.

So it will be really nice to have selectors to these components, so we can apply styles to all Tables or Forms, or the Sidebar container.

I'm not aware of any component that do that, there's a library that apply css selectors from Nova 3 to Nova 4 but in nova 4 still there'e other elemnts that need to be selected.

![Screenshot_18](https://user-images.githubusercontent.com/1440432/235205912-e835c02a-53e2-4202-b995-51b6f4ce8f80.png)
![Screenshot_19](https://user-images.githubusercontent.com/1440432/235205917-c4dfdf37-c063-4a41-8abd-30e7e5422ffb.png)
![Screenshot_20](https://user-images.githubusercontent.com/1440432/235205919-599e57cb-4305-4d9b-8d53-eb3445109d25.png)

**Comment by mahmoudegh** [comment]
I am also facing the same problem too.. I hope it will be fixed

**Comment by timoostrich** [comment]
Same problem here. I really need to add CSS rules to some image fields so SVGs are displayed correctly.
Someone has a good/easy solution for this?

**Comment by AbdallahAli912** [comment]
Same problem here.

**Comment by Afahmy9** [comment]
I'm facing the same issue as well

**Reply by HishamSeo** [reply]
same problem here please let me know if there is any solution for that

**Comment by heshammostafa123** [comment]
I'm facing the same issue

**Comment by Ahmed-Abdelhamee** [comment]
please fix this problem

**Comment by MostafaAmar** [comment]
Facing The same problem

**Comment by Mostafa-Mamduh** [comment]
is there any solution to this problem  ?

**Comment by Afahmy9** [comment]
Any updates on a solution?

**Comment by beshoo** [comment]
I'm facing the same issue

**Comment by tractorcow** [comment]
I noticed that in the recent update to nova (from 4.27 to 4.32) some existing classes (e.g. sidebar-item) were replaced in favour of pure tailwind. It's not just a problem, but it's one that's getting slowly worse. =(
